transit_odp/common/templatetags/filters.py

```diff
@@ -8,7 +8,21 @@
 
 @register.filter("percentage")
 def percentage(text, arg=-1):
-    return floatformat(text * 100.0, arg=arg) + "%"
+    try:
+        score = float(text)
+    except (TypeError, ValueError):
+        # Handle the case where the conversion to float fails (e.g., if value is not numeric)
+        return text
+
+    percentage_value = score * 100.0
+
+    # If value is 0.1 then 1% should be returned
+    # If value is 99.8 then 99% should be returned
+    if 0 < percentage_value < 1:
```

Comment:
@alvak1 You should only return these if int(arg==0), i.e.

if 0 < percentage_value < 1 and int(arg == 0):

if elif 99 < percentage_value < 100 and int(arg == 0):

Reply:
@PCridKPMG  done